src/net/sourceforge/kolmafia/KoLCharacter.java

```diff
@@ -4964,16 +4965,18 @@ public static final Modifiers recalculateAdjustments(
             && (offhand == null || offhand == EquipmentRequest.UNEQUIP);
 
     // Area-specific adjustments
-    newModifiers.add(Modifiers.getModifiers("Loc", Modifiers.currentLocation));
-    newModifiers.add(Modifiers.getModifiers("Zone", Modifiers.currentZone));
+    newModifiers.add(Modifiers.getModifiers(ModifierType.LOC, Modifiers.currentLocation));
+    newModifiers.add(Modifiers.getModifiers(ModifierType.ZONE, Modifiers.currentZone));
 
     // Look at sign-specific adjustments
-    newModifiers.add(Modifiers.MONSTER_LEVEL, MCD, "MCD:MCD");
-    newModifiers.add(Modifiers.getModifiers("Sign", KoLCharacter.ascensionSign.getName()));
+    newModifiers.addDouble(
+        Modifiers.MONSTER_LEVEL, MCD, ModifierType.MCD, "Monster Control Device");
```

Comment:
just a general comment on differences: I note that most of these are the same, some of these are slightly different (e.g. one is capitalised, like Brimstone or Cloathing), some are more different but understandably (e.g. MCD, quartet).

Most are capitalised, but some are not (e.g. fake hands). Are there intended to be rules as to how these are formed / capitalised?

Reply:
My rule was generally to capitalize unless it's the name of something, in which case use that capitalization. `fake hand` is a name, so I left it here.